test/unit/lib/fetcher.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('Fetcher', function () {
     request
       .withArgs({
         method: 'GET',
-        url: `/entries?limit=${Fetcher.perRequestLimit}&order=sys.createdAt&sys.archivedAt[exists]=false&sys.contentType.sys.id[in]=newsArticle&skip=0`
+        url: `/entries?limit=100&order=sys.createdAt&sys.archivedAt[exists]=false&sys.contentType.sys.id[in]=newsArticle&skip=0`
```

Comment:
I wonder if it makes sense to use a const variable for this instead of hard coding 100 in all these instances?

Reply:
@ruderngespra 
First I thought the same, but then I thought the rest of the test is also not parameterized.